Update faq to reflect the latest app update (#59)
Paperback-iOS/website: src/help/faq/README.md

If you are a patreon, you will recieve an email to your patreon address regarding information about accessing Paperback through TestFlight. The TestFlight version of the application contains many additional beta features which can be used before it becomes available on the App Store.

> **NOTE** 

Author: As there is more than one note, should a plural be used?
```suggestion
**NOTES** 
```

Reviewer: "Note" is is being used as a verb in this context for which there is no plural.

Author: It's interesting to know. Thanks for the review @GameFuzzy!

---

### How do I enable Hentai

>  * Sign into [MangaDex website](https://mangadex.org/) on a browser and go to your settings to set Hentai toggle to "**View All**".

Reviewer 2: ```suggestion
 * Sign into [MangaDex’s website](https://mangadex.org/) on a browse, go to your settings and set the Hentai toggle to "**View All**".
```

Author: What is a "browse" exactly. Could the `r` have unexpectedly disappear or was the modification intended?

Reviewer 2: Oops, sorry about that! I’ve adjusted the suggestion to fit the proper grammar

 * Then sign into MangaDex on Paperback, hitting "Remember me", and go into your *Content Settings* in the app settings tab and choose "**Show All**".

---

### My content settings are reset everytime I open the app
When you are logging in, make sure that **"Remember me"** is checked on the page.
Also make sure that your email address is verified on MangaDex.

---

### How can I add other manga sources?
Other manga sources are available on the latest TestFlight public and patreon versions of the application.
Read [Adding an External Repository](/help/guides/adding-repos/) guide to learn how to install other sources.

---

### I am on the public version of Paperback! When can I download manga to my device for offline viewing?
Downloading manga is available on TestFlight public and patreon versions of the application. Update to the latest TestFlight build to be able to download manga.

---

### How can I make a backup of my library?
**Creating a backup**
 * Go to settings > backup

>  * Press "**Create a backup**" and choose where you want to save the backup (Note is recommended)

Reviewer 2: ```suggestion
 * Press "**Create a backup**" and choose where you want to save the backup (We recommend you save them to your Notes app)
```

Reviewer 2: Or alternatively just call it Notes instead of Note

Author: Great improvement! Thanks

**Restoring a backup**

>  * Reinstall all sources used at the time of the backup

Reviewer 2: ```suggestion

```
I’ve updated the app so this step is automatic.

Reviewer 2: Note that this content might not have made it to the public version yet, so probably only applies to the patreon build

 * Go to your previously saved backup and press the share icon
 * Select "**Copy to Paperback**" action
 * You can then press "Restore" to restore your backup

**NOTE** 
 * Currently history tab is not saved in the backup process

>  * Manga won't be restored if their source is not installed

Reviewer 2: ```suggestion
 
```
This step isn’t needed in the patreon build of the app anymore. Maybe mark that somehow? It still probably applies for the public build

Author: Nice feature for the app, I'm sure it will make backup restore easier. 

Based on your suggestion, what would you think of removing `* Reinstall all sources used at the time of the backup` step and adding something like this to the **NOTE**:
```suggestion
 * On the public build, you need to manually reinstall each sources before restoring the backup, in order for their manga to be restored
```
```suggestion
 * Public build users need to manually reinstall each sources before restoring the backup, in order for their manga to be restored
```

Reviewer: ```suggestion
 * Public build users need to manually reinstall each source before restoring the backup in order for their manga to be restored
```
